package.json

```diff
@@ -83,5 +90,14 @@
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
   },
-  "dependencies": {}
+  "peerDependencies": {
```

Comment:
I think we miss some peer dependencies here:

`@openmrs/esm-error-handling`, `@openmrs/esm-api`